website/tosti/settings/base.py

```diff
@@ -166,6 +167,8 @@
 CONSTANCE_BACKEND = 'constance.backends.database.DatabaseBackend'
 
 CONSTANCE_CONFIG = {
+    'FOOTER_CREDITS_TEXT': ('TOSTI - Tartarus Order System for Take-away Items',
+                            'Text to display in the footer credits, specify [year] for the current year', str),
```

Comment:
How is [year] inserted? 
And does this also work with html?

Reply:
It does not work with HTML, [year] was something I thought of would be nice but let's not do it, it's not needed.